bool unexpected_object
float64 x
float64 y
float64 z
---
boot action_being_taken